Automattic/wp-calypso
# Stats: Refactoring controller and page files

Related to #87113

## Proposed Changes

* proposing refactoring of Stats controller and placement of the page components in order to centralise the main entry points and simplify the growing controller file

## Testing Instructions



* navigate to the live branch
* verify that `insights`, `subscribers` and `purchase` pages load correctly and you can interact with them in an expected way

## Pre-merge Checklist
